docs/source/release_procedure.org

```diff
@@ -0,0 +1,70 @@
+#+TITLE: Iroha v2 Release procedures
+#+AUTHOR: Aleksandr Petrosyan
+#+DATE: 2023-08-13
+* About
+
+This document outlines the procedures which I recommend following during the release of Iroha v2 release candidates and recommendations for how to adjust these procedures when the v2.0.0 is made public, announced, and subsequent semantically versions packages are released.
+
+
+* Timed release cadence
+
+Because Iroha does not have a limited scope or road-map of features that could serve as punctuation for the project, and oftentimes ideas are proposed during the development process, it was decided to release Iroha as is in a timed- rather than feature-gated- cadence. As a benefit, but not goal of this approach, the change synchronisation with SDKs happens in a bite-sized manner, leading to a more smooth overall transition and gradual synchronisation of ideas. It allows the core development team to remain agile and adaptable. 
+
+The timed cadence has a few disadvantages that could become relevant if the development pace slows down and features/fixes/optimisations become less prevalent than refactors and continuous integration improvements.
+
+However, it is my personal recommendation that the timed cadence be kept for as long as the core team retains the majority of its engineers in some capacity.
```

Comment:
let's refrain from subjective/personal statements as it's getting merged after review and considered to be in a more or less final shape 

```suggestion
However, it is recommended that the timed cadence be kept for as long as the core team retains the majority of its engineers in some capacity.
```